requirements.txt

```diff
@@ -7,7 +7,7 @@ msgpack>=0.5.6
 sortedcontainers
 uvloop>=0.14.0
 grpcio-tools>=1.33.2
-protobuf>=3.12.2,<4.0.0
+protobuf==4.23.4
```

Comment:
Fixing protobuf version to __latest__ can cause potential conflicts with some older projects. We gotta figure out some reasonably old version where the code (still) works.

Reply:
Please use `protobuf>=X`